<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CNC Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_TargetAddress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Target Address</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_IP">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.0.24</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_IP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Port">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>8000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Test">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_RemoteShell">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>511</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Remote Shell</string>
    </property>
    <widget class="QTextEdit" name="textEdit_Shell">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>491</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Enter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>350</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Command">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>371</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>ifconfig</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Upload">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Upload File</string>
    </property>
    <widget class="QPushButton" name="pushButton_Upload">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Download">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Download File</string>
    </property>
    <widget class="QPushButton" name="pushButton_Download">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Download">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
